src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceConditionService.kt

```diff
@@ -41,13 +46,37 @@ class LicenceConditionService(
     val entityStandardPssConditions = request.standardPssConditions.transformToEntityStandard(licenceEntity, "PSS")
 
     val username = SecurityContextHolder.getContext().authentication.name
+    val createdBy = communityOffenderManagerRepository.findByUsernameIgnoreCase(username)
```

Comment:
wondering about what happens if this is a team member who hasn't done licence stuff before - can we guarantee that they will exist in the DB?

Reply:
This is probably fine - just requires a test

Reply:
Added an extra test